master/internal/workspace/postgres_workspace.go

```diff
@@ -164,3 +180,36 @@ func GetAllNamespacesForRM(ctx context.Context, rmName string) ([]string, error)
 	}
 	return ns, nil
 }
+
+// AddWorkspaceNamespaceBinding adds a workspace-namespace binding.
+func AddWorkspaceNamespaceBinding(ctx context.Context, tx *bun.Tx,
```

Comment:
nit: in our codebase, the naming convention for db funcs that pass in a transaction is to add the suffix "Tx" at the end

Reply:
Changed